package.json

```diff
@@ -74,7 +74,9 @@
     "test:browser": "grunt && grunt karma:main",
     "test:esmodules": "grunt && grunt karma:esmodules",
     "test:amd": "grunt && grunt karma:amd",
-    "test": "grunt && grunt test:slow && grunt karma:main && grunt karma:esmodules && grunt karma:amd",
+    "test:no-deprecated": "grunt test:prepare && grunt custom:-deprecated && grunt karma:main",
+    "test:slim": "grunt test:prepare && grunt custom:-ajax,-effects && grunt karma:main",
```

Comment:
You probably already planned to do this, but change to `custom:slim` when that's merged.

Reply:
@timmywil PR updated, I rebased it & added a new commit modifying this line.